src/app/pages/FastBtcPage/components/Withdraw/AmountForm.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useCallback, useContext, useMemo, useState } from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+import { bignumber } from 'mathjs';
+import { AssetSymbolRenderer } from '../../../../components/AssetSymbolRenderer';
+import { Asset } from '../../../../../types';
+import { translations } from 'locales/i18n';
+import { WithdrawContext, WithdrawStep } from '../../contexts/withdraw-context';
+import { useBalance } from '../../../../hooks/useBalance';
+import { AmountInput } from '../../../../components/Form/AmountInput';
+import { FormGroup } from 'app/components/Form/FormGroup';
+import { gasLimit } from '../../../../../utils/classifiers';
+import { TxType } from '../../../../../store/global/transactions-store/types';
+import { WithdrawDetails } from './WithdrawDetails';
+import { FastBtcButton } from '../FastBtcButton';
+import { weiToNumberFormat } from '../../../../../utils/display-text/format';
+import { LoadableValue } from '../../../../components/LoadableValue';
+import { NetworkAwareComponentProps } from '../../types';
+import { getBTCAssetForNetwork } from '../../helpers';
+import { btcInSatoshis } from 'app/constants';
+
+export const AmountForm: React.FC<NetworkAwareComponentProps> = ({
+  network,
+}) => {
+  const { amount, limits, set } = useContext(WithdrawContext);
+  const { t } = useTranslation();
+
+  const balance = useBalance();
+
+  const [value, setValue] = useState(amount);
+
+  const invalid = useMemo(() => {
+    const amount = value || '0';
```

Comment:
Do you need this? Because `amount` is `string` so it shouldn't be `null` or `undefined` (unless set incorrectly somewhere but it would be 'null' or 'undefined' so this condition would still return `value` and not `0`).